scania-digital-design-system/tegel
# refactor(banner): variant instead of type prop

**Describe pull-request**  
Rename prop type to variant

**Solving issue**  
Fixes: [DTS-2130](https://tegel.atlassian.net/browse/DTS-2130)

**How to test**  
1. Go to Netlify preview link
2. Check if Banner component updates styling and code with renamed prop


**Suggested test steps**




[DTS-2130]: https://tegel.atlassian.net/browse/DTS-2130?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
